ZelBack/src/services/appsService.js

```diff
@@ -1859,23 +1859,30 @@ async function createAppVolume(appSpecifications, appName, isComponent, res) {
     }
 
     // if s flag create .stfolder
-    const containerDataFlags = appSpecifications.containerData.split('|')[0].split(':')[1] ? appSpecifications.containerData.split('|')[0].split(':')[0] : '';
-    if (containerDataFlags.includes('s')) {
-      const stFolderCreation = {
-        status: 'Creating .stfolder for syncthing...',
-      };
-      log.info(stFolderCreation);
-      if (res) {
-        res.write(serviceHelper.ensureString(stFolderCreation));
-      }
-      const execDIRst = `sudo mkdir -p ${appsFolder + appId}/.stfolder`;
-      await cmdAsync(execDIRst);
-      const stFolderCreation2 = {
-        status: '.stfolder created',
-      };
-      log.info(stFolderCreation2);
-      if (res) {
-        res.write(serviceHelper.ensureString(stFolderCreation2));
+    const containersData = appSpecifications.containerData.split('|');
```

Comment:
Can we write some manual tests for this? Like how does it work and what all combinations are now possible?

Reply:
It works exactly the same way as it was announced on this version -> https://github.com/RunOnFlux/flux/pull/1141
Simply wasn't working properly, from the tests Tom W. was doing with Minecraft app, we figured out things we were missing on fluxOs that are now working as intended.